ui-src/src/components/LeaderBoard.tsx

```diff
@@ -13,36 +13,34 @@ type LeaderBoardProps = {
 
 class LeaderBoard extends React.Component<LeaderBoardProps, {}> {
 
+
+
   public render () {
     // Fetch the score
     const {scores, allPlayerHandles} = this.props
 
-    const tempUserScoreMap:any[] = []
     const displayBoard:any[] = []
 
-    // Fetch the identity
     if(scores !== null && allPlayerHandles.size !== 0) {
-      scores.forEach((agentScore:number, agentHash:Hash) => {
-
+      const descending = (a:number, b:number) => {
+        if (a < b) { return 1; }
+        if (a > b) { return -1; }
+        if (a === b) { return 0; }
+        return 0;
+      }
+
+      scores.sort(descending).forEach((agentScore:number, agentHash:Hash) => {
```

Comment:
I think you could shorten this to scores.sort((a, b) => b-a) but anyway. If it works it works